packages/kbn-apm-synthtrace-client/src/lib/infra/host.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import { Entity, Fields } from '../entity';
+import { Serializable } from '../serializable';
+
+interface HostDocument extends Fields {
+  'host.hostname': string;
+}
+
+export class Host extends Entity<HostDocument> {
+  metrics() {
```

Comment:
should we name this `cpuMetrics()` or `systemCpu()` as we may want to implement memory/network.. metrics in the future ?

Reply:
I had it as systemMetrics first but then I just wanted to make it simple since it's not really representative of that metricset either yet. 
Let's decide on that in the other PR!